GoogleForCreators/web-stories-wp
# Update templates to use TT for uppercase text

## Context

With the addition of uppercase toggle for text, template JSON needed to be updated to use TT.

## Summary

Updates templates to use `text-transform` to make text uppercase, for following templates:

- Fresh & Bright
- Food & Stuff
- Doers Get More Done
- Weekly Entertainment
- Fashion On The Go
- No Days Off
- Experience Thailand
- Sleep
- Baking Bread Guide
- Trendy Winter Veggie
- Ways to Eat Avocado
- Kitchen Stories
- Album Releases

## Testing Instructions

- Go to Explore Templates screen, path `wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery`
- Pick any one from the above listed templates and create a new story by clicking `Use Template` button.
- Select an uppercase word, observe that the `TT` button is highlighted in the settings panel.

## Checklist




---

Fixes #7658
